app/assets/javascripts/controllers/middleware_topology/middleware_topology_controller.js

```diff
@@ -10,7 +10,7 @@ function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologySer
 
     $scope.refresh = function() {
         var id;
-        if ($location.absUrl().match("show/$") || $location.absUrl().match("show$")) {
+        if ($location.absUrl().match('show/$') || $location.absUrl().match('show$')) {
```

Comment:
those kind of changes make the review harder as they are not related to the PR itself. please try to separate those fixes in the future to another PR.